src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -128,16 +147,46 @@ public XMLStructuredOutput getOutput() throws IOException {
         return new XMLStructuredOutput(outcall.postFromOutput());
     }
 
+    private InputStream getInputStream() throws IOException {
+        if (logger != null && logger.isFINE()) {
+            // log the request, even when parsing fails
+            try (InputStream body = outcall.getResponse().body()) {
+                String text = new String(body.readAllBytes(), StandardCharsets.UTF_8);
```

Comment:
we need to use the xml parse + transform here as the charset can by anything....

Reply:
I would very much like to avoid this - if the response cannot be parsed, nothing can be logged, but these are exactly the cases where a logging would be very helpful

Reply:
Please have a look at the updated code. I use the encoding suggested by the outcall, and the encoding now only effects the log, not the returned stream